bindings/mcms_encoder.go

```diff
@@ -0,0 +1,324 @@
+		if err != nil {
+			return nil, err
+		}
+		switch function {
+		case "accept_ownership":
+			// TODO: Find correct type args
+			typeArgs := []string{"0x1::sui::SUI"}
+			return lockReleaseTokenPool.Encoder().McmsAcceptOwnershipWithArgs(typeArgs, stateObj, executingCallbackParams)
+		case "set_rebalancer":
+		case "set_allowlist_enabled":
+		case "apply_allowlist_updates":
+		case "apply_chain_updates":
+		case "add_remote_pool":
+		case "remove_remote_pool":
+		case "transfer_ownership":
+		case "execute_ownership_transfer":
+			return encodeDefaultWithTypeArgs()
+		case "set_chain_rate_limiter_configs":
+		case "set_chain_rate_limiter_config":
+			return encodeDefaultWithTypeArgsAndClock()
+
+		}
+
+	// MANAGED TOKEN POOL
+	case "managed_token_pool":
+		managedTokenPool, err := module_managed_token_pool.NewManagedTokenPool(target, e.client)
+		if err != nil {
+			return nil, err
+		}
+		switch function {
+		case "accept_ownership":
+			// TODO: Find correct type args
+			typeArgs := []string{"0x1::sui::SUI"}
+			return managedTokenPool.Encoder().McmsAcceptOwnershipWithArgs(typeArgs, stateObj, executingCallbackParams)
+		case "set_allowlist_enabled":
+		case "apply_allowlist_updates":
+		case "apply_chain_updates":
+		case "add_remote_pool":
+		case "remove_remote_pool":
+		case "transfer_ownership":
+		case "execute_ownership_transfer":
+			return encodeDefaultWithTypeArgs()
+		case "set_chain_rate_limiter_configs":
+		case "set_chain_rate_limiter_config":
+			return encodeDefaultWithTypeArgsAndClock()
+		}
+
+	// USDC TOKEN POOL
+	case "usdc_token_pool":
+		usdcTokenPool, err := module_usdc_token_pool.NewUsdcTokenPool(target, e.client)
+		if err != nil {
+			return nil, err
+		}
+		switch function {
+		case "accept_ownership":
+			// TODO: Find correct type args
+			typeArgs := []string{"0x1::sui::SUI"}
+			return usdcTokenPool.Encoder().McmsAcceptOwnershipWithArgs(typeArgs, stateObj, executingCallbackParams)
+		case "set_allowlist_enabled":
+		case "apply_allowlist_updates":
+		case "apply_chain_updates":
+		case "add_remote_pool":
+		case "remove_remote_pool":
+		case "transfer_ownership":
+		case "execute_ownership_transfer":
+			return encodeDefaultWithTypeArgs()
+		case "set_chain_rate_limiter_configs":
+		case "set_chain_rate_limiter_config":
+			return encodeDefaultWithTypeArgsAndClock()
+		}
+
+	// MANAGED TOKEN
+	case "managed_token":
+		managedToken, err := module_managed_token.NewManagedToken(target, e.client)
+		if err != nil {
+			return nil, err
+		}
+		switch function {
+		case "accept_ownership":
+			// TODO: Find correct type args
+			typeArgs := []string{"0x1::sui::SUI"}
+			return managedToken.Encoder().McmsAcceptOwnershipWithArgs(typeArgs, stateObj, executingCallbackParams)
+		case "configure_new_minter":
+			// TODO: Find correct type args
+			typeArgs := []string{"0x1::sui::SUI"}
+			// TODO: This doesn't exist
+			denyList := bind.Object{Id: "0xdef"}
```

Comment:
Validating `deny_list` can be removed, it's not needed as it's not passed to the underlying function.